yoti-sdk-auth/src/main/java/com/yoti/auth/FormRequestClient.java

```diff
@@ -0,0 +1,102 @@
+package com.yoti.auth;
+
+import static com.yoti.api.client.spi.remote.call.YotiConstants.DEFAULT_CHARSET;
+
+import java.io.ByteArrayOutputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UnsupportedEncodingException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import java.util.Scanner;
+
+import com.yoti.api.client.spi.remote.call.ResourceException;
+import com.yoti.api.client.spi.remote.util.QuietCloseable;
+
+/**
+ * Internal use only.
+ *
+ * The {@link FormRequestClient} is used for performing an application/x-www-form-urlencoded
+ * HTTP request using base Java libraries only.
+ */
+final class FormRequestClient {
+
+    String performRequest(String apiUrl, String method, Map<String, String> formParams) throws IOException, ResourceException {
+        String urlParams = getDataString(formParams);
+
+        byte[] postData = urlParams.getBytes(StandardCharsets.UTF_8);
```

Comment:
Why return `String` and then do `getBytes(...)` rather than simply return `byte[]` from the get go? `urlParams` doesn't seem to be used anywhere else and what this (method) seems to need is the `byte[]` not `String`